chrome-launcher.ts

```diff
@@ -121,12 +122,13 @@ export class Launcher {
   }
 
   private get flags() {
-    let flags = DEFAULT_FLAGS.concat([
-      `--remote-debugging-port=${this.port}`,
+    let flags = DEFAULT_FLAGS.concat([`--remote-debugging-port=${this.port}`]);
+
+    if (!this.useDefaultProfile) {
       // Place Chrome profile in a custom location we'll rm -rf later
```

Comment:
let's maybe move this line of the comment down to where we create it with a bit of explanation on the new logic

Reply:
Done